packages/components/accordion/src/accordion.ts

```diff
@@ -0,0 +1,63 @@
+import { type ScopedElementsMap, ScopedElementsMixin } from '@open-wc/scoped-elements/lit-element.js';
+import { type CSSResultGroup, LitElement, type PropertyValues, type TemplateResult, html } from 'lit';
+import { property, queryAssignedElements } from 'lit/decorators.js';
+import { AccordionItem } from './accordion-item.js';
+import styles from './accordion.scss.js';
+
+/**
+ * An accordion component that can contain accordion-items
+ *
+ * ```html
+ *   <sl-accordionn>...</sl-accordion>
+ * ```
+ *
+ * @slot default - The place for multiple <sl-accordion-item>
+ */
+export class Accordion extends ScopedElementsMixin(LitElement) {
+  /** @private */
+  static get scopedElements(): ScopedElementsMap {
+    return {
+      'sl-accordion-item': AccordionItem
+    };
+  }
+
+  /** @private */
+  static override styles: CSSResultGroup = styles;
+
+  /** Whether only one accordion item can be opened at once. By default, multiple accordion items can be opened. */
+  @property({ type: Boolean, reflect: true }) single?: boolean;
+
+  /** @private The slotted accordion items. */
+  @queryAssignedElements() items!: AccordionItem[];
+
+  override updated(changes: PropertyValues<this>): void {
+    super.updated(changes);
+
+    if (changes.get('single') === false) {
+      this.items.forEach(item => {
+        if (item.renderRoot.querySelector('details')?.hasAttribute('open')) {
+          item.renderRoot.querySelector('summary')?.click();
+        }
+      });
+    }
+  }
+
+  override render(): TemplateResult {
+    return html` <slot @click=${this.#onClick}></slot> `;
+  }
+
+  #onClick(event: Event): void {
+    if (!this.single || event.defaultPrevented) {
+      // No toggling when no `single` or the user prevents it.
+      return;
+    }
+
+    this.items.forEach(item => {
+      if (item !== event.target) {
```

Comment:
This could probably be a `.filter(item => item !== event.target)` instead of an if statement?